source/includes/_capture_external_event_api.md

```diff
@@ -0,0 +1,72 @@
+# Capture External Event API
```

Comment:
I feel that the word API is redundant here.  Consider Capture External Events

Reply:
I will remove API